packages/client/src/lobby/createReplayJSON.ts

```diff
@@ -50,23 +49,57 @@ export default function createJSONFromReplay(room: string) {
   game.actions = getGameActionsFromState(replay);
 
   // Add the hypothesis from log, after current segment
-  const states = replay.hypothetical!.states;
-  const log = states[states.length - 1].log;
-  if (replay.segment < log.length) {
-    const slice = log.slice(replay.segment + 1);
-    const actions = getGameActionsFromLog(slice);
-    game.actions.push(...actions);
+  if (replay.hypothetical !== null) {
+    const states = replay.hypothetical.states;
```

Comment:
i think destructuring would be better here

Reply:
not sure what to do

Reply:
`const { states } = replay.hypothetical;`

Reply:
the point is that you avoid typing the word states twice, which wastes time + creates more code